src/utils/create-transform.ts

```diff
@@ -31,13 +34,19 @@ export function create_transform(transformer: Transformer) {
       const output_file: gulp_util.File = input_file.clone();
       try {
```

Comment:
The `try...catch` seems unnecessary here since it's used to catch errors from `transformer`.